pkg/codegen/python/utilities.py

```diff
@@ -280,6 +280,18 @@ def call_plain(
 
     return result
 
```

Comment:
@Frassle is all this code missing package references? Do we have to add some stuff here (OK with doing this in a follow-up but it looks like an omission to me)